cmd/wallet-web/src/pages/Vaults.vue

```diff
@@ -9,22 +9,29 @@
     <div class="mb-8 w-full">
       <h3 class="text-neutrals-dark">{{ $t('Vaults.heading') }}</h3>
     </div>
-    <div class="px-6 w-64 h-auto bg-neutrals-white rounded-xl">
-      <div class="pt-5">
-        <div class="flex justify-center items-center w-12 h-12 bg-gradient-pink rounded-full">
-          <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
-        </div>
-      </div>
-      <div class="pb-4 space-y-1">
-        <span class="block pt-4 text-lg font-bold"> {{ $t('Vaults.allVaults') }}</span>
-        <span class="block text-sm text-neutrals-medium">
-          {{
-            $tc('Vaults.foundCredentials', credentialsFound, {
-              credentialLength: credentialsFound,
-            })
-          }}
-        </span>
-      </div>
+    <div class="grid grid-cols-1 lg:grid-cols-3 gap-4 xl:gap-8">
+      <vault-card
+        color="pink"
+        type="regular"
+        :credential-found="
+          $tc('Vaults.foundCredentials', credentialsFound, {
+            credentialLength: credentialsFound,
+          })
+        "
+        :text="$t('Vaults.allVaults')"
+      />
+      <!--Issue-1198 Add flyout menu to default and otther vaults -->
+      <vault-card
+        color="purple"
+        type="regular"
+        :credential-found="
+          $tc('Vaults.foundCredentials', credentialsFound, {
+            credentialLength: credentialsFound,
+          })
+        "
+        :text="$t('Vaults.defaultVault')"
```

Comment:
Is this a `regular` type or a `default`? I understand that the only difference will be in the flyout menu for the default vault (we won't show the delete button), but I guess it would be logical if we passed this down from here

Reply:
I will address this after #issue-1184 gets merged, we can discuss as needed then